### DISCUSSION TITLE
E5+ monolithic main body

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#43] [Ideas]
![image](https://user-images.githubusercontent.com/111667850/195720653-2d7f8b54-820c-45b6-93f3-ef9c59ad893d.png)

e5+ can do it all in one go to make it seamless. did some mild editing around the accessory card and fan vent area.

**Comment by jon-harper** [comment]
Do you mind if I take a look at the file? I'm curious about your changes. 😃

Also I'm guessing that your E5+ is back up, then? That's good news!

**Reply by TheoBromine2022** [reply]
So I took the holes out of the area around the fan vent and lowered the seam between the two pieces. Also all of the screw holes are solid. Yeah once I figure out what kind of file I can upload I'll send it to you.

**Reply by jon-harper** [reply]
STL will work fine for me. I just want a look-see. 🙂

**Reply by TheoBromine2022** [reply]
I cant upload STL's to github on these boards (or, if I can, I do not know how) so I will post this on thingiverse and de-list it later today

https://www.thingiverse.com/thing:5565392

**Reply by jon-harper** [reply]
Got it. Copy/paste or drag-n-drop in the comment box should let you upload files, btw.

**Comment by jon-harper** [comment]
By the way, yesterday I was thinking about your suggestion awhile back to unify the base.

I'm pretty far along with the next release, so I'm testing out new ideas. The work I did before last release to reorganize the source files makes it fairly easy to maintain and release a new variation. There will definitely be a unified base in the next release (the STL is ready to be committed), and possibly a "stock" main body variation or two.

**Reply by TheoBromine2022** [reply]
![image](https://user-images.githubusercontent.com/111667850/195864265-b1261bc9-c405-493e-a133-f7795193eedf.png)


So there's the base. I intend on putting filters in the fan vents so I felt like the additional holes in the front, while visually appealing, were just ways for dust to get in. I flattened the ridges on the bottom as well so it sits flush on the surface it sits on and removed the large rectangular interfaces for the joining screws. Im debating if I want to add a floor to this to seal the bottom from dust.

also, no, my e5+ is still not printing these but once I get this .98 omnibox at barebones state I will turn around and make these.

**Reply by jon-harper** [reply]
> I intend on putting filters in the fan vents so I felt like the additional holes in the front, while visually appealing, were just ways for dust to get in.

Good point. The new version has three vents and a fan mount behind each. There should be some kind of dust filter that size that could be used even when there's no fan. I'll also take that into consideration with the intakes in the main body...

>  Im debating if I want to add a floor to this to seal the bottom from dust.

I haven't done it yet, but a user on Reddit pointed out the risk of small kids lifting the box and doing bad things. They suggested just a terminal cover, but I'm going to use the screw holes on the base to make a two-piece bottom plate for safety and cover the whole thing. I'll make versions with small holes for speedy print and one that's solid for a closed fit. It won't be official until the next release, but I can upload it here.

**Reply by jon-harper** [reply]
[Bottom Panel - Closed.zip](https://github.com/jon-harper/OmniBox/files/9787512/Bottom.Panel.-.Closed.zip)

Interesting, I guess I can't upload an STL. Here they are as a zip. It's not fit tested yet of course, but should work.

**Reply by TheoBromine2022** [reply]
that looks pretty reasonable. you already have holes going all the way through so this should be an easy retrofit. im debating if I just want to add the floor to the bottom of the base as one giant print.

**Comment by jon-harper** [comment]
I would recommend against, as removing the PSU will become insanely difficult. OTOH, I just realized that I didn’t account for the heads of the screws. =/ I need to do a proper test print instead of dashing stuff off. My apologies…

Edit: so replying to email notifications needs work...FWIW, here's a fixed version. You can also find the files in the 0.9.9-experimental branch under [Panels/Bottom Panel](https://github.com/jon-harper/OmniBox/tree/0.9.9-experimental/Panels/Bottom%20Panel).

[Bottom Panel - Closed (Fixed).zip](https://github.com/jon-harper/OmniBox/files/9788068/Bottom.Panel.-.Closed.Fixed.zip)

**Reply by TheoBromine2022** [reply]
if you print it with the intention of flipping the print upside down you could add penny/dime sized circles where the screws go to give it "feet" and have recessions for the heads of the machine screw. it would give it a classy look and you could make the top of the omnibox have those same circles as indentations so they fit together. one thing i realized is that with the 500w psu it will require a thicker base to have the floorplate installed so I will keep that in mind on my final monolithic edits.

**Reply by TheoBromine2022** [reply]
[monolithic base.zip](https://github.com/jon-harper/OmniBox/files/9788260/monolithic.base.zip)

**Reply by jon-harper** [reply]
Hmm. I like the idea. I'll experiment some more.

Thanks for reminding me of that. I have base extensions already, but I'll need to turn them into more like a base "shim" that sits between the bottom panel and the base.
